nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -308,6 +314,36 @@ private UpdateableReference loadReference(String requestedRef) {
     }
   }
 
+  private RefreshableReference loadReferenceAtTime(String requestedRef, Instant timestamp) {
+    try {
+      Reference ref = requestedRef == null ? client.getTreeApi().getDefaultBranch()
+          : client.getTreeApi().getReferenceByName(requestedRef);
+      if (ref instanceof Hash) {
+        throw new IllegalArgumentException(String.format("Cannot specify a hash %s and timestamp %s together. " +
+            "The timestamp is redundant and has been ignored", requestedRef, timestamp));
+      }
```

Comment:
nit: add space after `if` and `for`

Reply:
I am not 100% sure I take your meaning. I have added a newline after the `if` and `for`, is that what you meant?

Reply:
yes, thank you